Added Media section permission for anonymous user to be able to visua… (#1862)
ezsystems/developer-documentation: docs/tutorials/beginner_tutorial/5_display_a_list_of_content_items.md

    </td>
</tr>
```
### Add Media permission

> To be able to visualise the `photo` field you have to add `read` permission to `Media` section.

Reviewer: ```suggestion
To be able to view the `photo` Field you have to add a `read` permission to `Media` section.
```

> From **Admin** (gear icon) in the left menu, go to the **Roles** management screen and click on the **Anonymous** Role.

Reviewer: ```suggestion
From **Admin** (gear icon) in the left menu, go to the **Roles** management screen and click the **Anonymous** Role.
```

![Policies for the Anonymous Role without Media section](step5_admin_anonymous_policies_without_media_section.png)

> Edit the **Content/Read** policy line to add `Media` section to **Limitation** along with the `Standard` section.

Reviewer: ```suggestion
Edit the **Content/Read** Policy line to add the `Media` section to **Limitation** along with the `Standard` section.
```
